nowledge-mem-codex-plugin/tests/test_codex_plugin.py

```
@@ -0,0 +1,548 @@
+
+    def test_main_counts_generic_refresh_errors(self):
+        temp_dir = tempfile.TemporaryDirectory()
+        self.addCleanup(temp_dir.cleanup)
+        rollout_path = Path(temp_dir.name) / "rollout-1.jsonl"
+        rollout_path.write_text("placeholder", encoding="utf-8")
+
+        self.module.parse_codex_session_streaming = mock.Mock(
+            return_value={
+                "thread_id": "codex-thread-1",
+                "messages": [{"role": "user", "content": "real request"}],
+                "workspace": "/tmp/project",
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/project"

(S108)